ports/servoshell/prefs.rs

```diff
@@ -90,9 +92,12 @@ pub fn default_config_dir() -> Option<PathBuf> {
     Some(config_dir)
 }
 
+/// Overrides the default preference dir
+#[cfg(any(target_os = "android", target_env = "ohos"))]
+pub(crate) static OVERRIDE_DEFAULT_PREFS_DIR: OnceLock<PathBuf> = OnceLock::new();
```

Comment:
It's used in default_config_dir(), so probably this variable should also be called default config dir.